frontend/src/__tests__/cypress/cypress/support/e2e.ts

```diff
@@ -27,6 +27,19 @@ Cypress.Keyboard.defaults({
   keystrokeDelay: 0,
 });
 
+if (Cypress.env('MOCK')) {
+  Cypress.on('uncaught:exception', (error) => {
+    // Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'https://cdn.jsdelivr.net/npm/monaco-editor@0.43.0/min/vs/base/worker/workerMain.js' failed to load.
+    if (
+      error.message.includes("Failed to execute 'importScripts' on 'WorkerGlobalScope'") &&
+      error.message.includes('monaco-editor')
+    ) {
+      return false;
+    }
+    return undefined;
+  });
+}
```

Comment:
This will fix this issue as well: https://issues.redhat.com/browse/RHOAIENG-7650